build/windows/installer/conf/fluent-bit.conf

```diff
@@ -2,15 +2,15 @@
     Flush        15
     Daemon       Off
     Log_Level    info
-    Log_File      /etc/fluent-bit/fluent-bit.log
+    Log_File     /etc/fluent-bit/fluent-bit.log
 
 [INPUT]
-    Name            forward
-    Listen          127.0.0.1
-    Port            25230
-    Mem_Buf_Limit   10m
-    Chunk_Size      32
```

Comment:
The naming and units for these are different and were causing errors for the newer fluent-bit version: https://docs.fluentbit.io/manual/pipeline/inputs/forward